.github/workflows/release-brew.yaml

```diff
@@ -161,10 +159,6 @@ jobs:
       with:
         pattern: bottle-*
 
-    - name: Set up Homebrew
```

Comment:
Have you confirmed this action gives no added value?

Reply:
We didn't have this before, so that's why I am now removing it. I only added this in one of my many debugging attempts.